examples/example_analysis_workflows.py

```diff
@@ -0,0 +1,197 @@
+
+    TODO:
+    - When is data discarded from memory?
+    - Should the observation grouping specification happen here,
+      so that the obs grouping code isn't duplicated for the various analyses?
+    """
+    from gammapy.data import DataStore, ObservationList
+    data_store = DataStore.from_dir(dir=...)
+    obs_id = data_store.obs_table.cone_select(...)
+
+    observations = ObservationList(data_store=..., obs_id=...)
+    observations = data_store.make_obs_list(selection=...)
+
+    return observations
+
+
+def run_scalar_analysis():
+    """Run 0-dim scalar analysis.
+
+    By this we mean get scalar numbers for simple stats for a given
+    region and energy band.
+
+    Goals:
+    - per-run excess, significance, livetime, ...
+    - total excess, significance, livetime, ...
+    """
+    from gammapy.data import RegionBackgroundEstimator, BackgroundEstimate
+
+
+def run_spectral_analysis():
+    """Run 1-dim spectral analysis.
+
+    Goals:
+    - counts and exposure vectors
+    - best-fit model values
+    - joint fit on observations or stacked analysis
+    - nice to have: access total stats, e.g. total significance
+    - nice to have: access run-wise results
+    """
+    # Note: `SpectrumMaker` is currently called `SpectrumExtraction`
+    from gammapy.spectrum import SpectrumMaker, SpectrumFit, SpectrumFitResult, SpectrumGrouping, \
+        SpectrumObservation, SpectrumObservationList, SpectrumResult, SpectrumResultDict, SpectrumStats
+    from gammapy.spectrum import PowerLaw
+    from gammapy.data import Target, TargetSummary, DataStore
+    from gammapy.data import RegionBackgroundEstimator, BackgroundEstimate
+
+    observations = get_observations()
+    target = Target(pos='dummy', on_region='dummy')
+    config = dict(method='dummy', apply_psf_correction=True)
+
+    # User can run this explicitly, can be done via spectrum_extraction.estimate_bg()
+    # bg_estimator = BackgroundEstimator(observations, target, **config)
+    # bg_estimator.run() # computes reflected regions and filters off lists
+    #
+    # # If you want, the `BackgroundEstimate` is serialisable
+    # bg_estimator.background_estimate.write(dir='temp')
+    # bg_estimate = BackgroundEstimate.read(dir='temp')
+
+    spectrum_maker = SpectrumMaker(bg_estimate, observations, target, **config)
+    spectrum_maker.run()
+
+    # spectrum_extraction.make_reflected_regions()
+    # spectrum_extraction.bg_estimate
+    # spectrum_extraction.estimate_background()
+    # stats_table = spectrum_extraction.make_stats_table()
+
+    # If you want, the `SpectrumObservationList` is serialisable
+    spectrum_maker.spectrum_observation_list.write(dir='temp')
+    spectrum_obs_list = SpectrumObservationList.read(dir='temp')
+
+    # Compute new stacked spectra with a given grouping
+    group_id = 'left up to the user for now'  # Python list of group_id for each obs_id
+    spectrum_obs_list = spectrum_obs_list.apply_stacking(group_id=group_id)
```

Comment:
That is already the case, isn't it?
You take the ``ObservationTable`` from the store (maybe add an ``OFFSET`` column, for a certain target position) and run the observation grouping functionality (the generic stuff in ``gammapy.data``). 

At this point in the analysis the extracted 1D spectra have to be stacked in the correct groups (you cannot to this before the analysis, because you have to extract them first). I think the API is best like it is done in ``astropy.table.vstack``
```
SpectrumObservation = stack(SpectrumObservationList)

```